common/types.go

```diff
@@ -54,6 +54,8 @@ var (
 // Hash represents the 32 byte Keccak256 hash of arbitrary data.
 type Hash [HashLength]byte
 
+type BlockNumber int64
+
```

Comment:
Is it duplicate with `rpc.BlockNumber`? `common.BlockNumber` is not used anywhere.
Since many codes already use `rpc.BlockNumber`, perhaps `common.BlockNumber` is not necessary.

Reply:
Thanks, reflected in [9da03ac](https://github.com/klaytn/klaytn/pull/1170/commits/9da03ace27dc8016a0016c629c6ef5073c71596d)